webapp/routes/route_team.jsx

```diff
@@ -312,6 +321,19 @@ export default {
                             System.import('components/sidebar.jsx'),
                             System.import('components/permalink_view.jsx')
                         ]).then(
+                            (comarr) => callback(null, {team_sidebar: comarr[0].default, sidebar: comarr[1].default, center: comarr[2].default})
+                        );
+                    }
+                },
+                {
+                    path: 'channel/:channelId/pl/:postid',
+                    onEnter: onPermalinkEnter,
```

Comment:
Note that this calls the same function as when you hit `/pl/:postid` above. The difference is that this "overload" includes a channel id, so it adds the channel to your LHS. The "overload" is used by the jump links in the RHS. Not sure if this is the best way to solve this problem.